nixos/doc/manual/from_md/release-notes/rl-2111.section.xml

```diff
@@ -75,7 +75,13 @@
   </section>
   <section xml:id="other-notable-changes">
     <title>Other Notable Changes</title>
-    <para>
-    </para>
+    <itemizedlist spacing="compact">
+      <listitem>
+        <para>
+          The <literal>bitwarden_rs</literal> packages and modules were
```

Comment:
Would the present state of the code would cause this service to fail for existing users because of ownership permissions?

Reply:
Nope, systemd will automatically change the ownership of `StateDirectory` to the user that owns the service, so we're good here.

Reply:
Ignore this, see above comment.